words/predicates.txt

```diff
@@ -1003,7 +1162,9 @@ sky
 slash
 sleepy
 sleet
+slender
 slime
+slinky
```

Comment:
Aside from the toy I don't know if I've _ever_ heard this used without sexual overtones.

Reply:
loooool